.github/workflows/jacoco_check.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Add coverage to PR
         if: steps.jacocorun.outcome == 'success'
         id: jacoco-model
-        uses: madrapps/jacoco-report@${{ env.jaCocoReportVersion }}
+        uses: madrapps/jacoco-report@v1.6.1
```

Comment:
Why switch back to hardcoded value?

Reply:
Yeah, this is an error. Fixed in commit 297057f353d2d1c7690637cb175508d48b91ac44.

Reply:
The latest change introduced an error. I was trying to fix it. I could not find how to provide the version using the env variable in the uses line.

Reply:
> https://github.com/orgs/community/discussions/25246#discussioncomment-3247073

Won't work

Reply:
I kept hardcoded values as thay are used everywhere. I do not see a reason why this GH action should be solved in a different way.